docs/src/components/HomepageFeatures.module.css

```diff
@@ -0,0 +1,13 @@
+/* stylelint-disable docusaurus/copyright-header */
+
+.features {
+  display: flex;
+  align-items: center;
+  padding: 2rem 0;
+  width: 100%;
+}
+
+.featureSvg {
```

Comment:
```suggestion
.feature-svg {
```
css convention

Reply:
I think I tried, no r sure this plug-in translates